### PR TITLE
FIX: Improve broadcasting experience #66

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: screen/send/success.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            width: 120,
+            height: 120,
+            borderRadius: 60,
+            alignSelf: 'center',
+            justifyContent: 'center',
+            marginTop: 43,
+            marginBottom: 53,
+          }}
+        >
+          <Image style={{ alignSelf: 'center' }} source={require('../../img/baseline-check-24-px.png')} />
```
Comment (Overtorment) [inline screen/send/success.js]: Id prefer to use font-awesome or some other icon already bundled

File: screen/send/details.js [hunk 2]
```diff
@@ -538,7 +544,8 @@ SendDetails.propTypes = {
       params: PropTypes.shape({
         address: PropTypes.string,
         fromAddress: PropTypes.string,
-        fromSecret: PropTypes.string,
+        satoshiPerByte: PropTypes.string,
+        fromSecret: PropTypes.fromSecret,
```
Comment (Overtorment) [inline screen/send/details.js]: broadcast() is now duplicated in 2 screens. lets completely remove broadcast functionality from "Transaction details" and leave it only on `send/confirm` screen. will be easier to maintain and look more natural

File: screen/send/confirm.js [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    headerTintColor: '#0c2550',
+  };
+
+  constructor(props) {
+    super(props);
+    console.log('send/create constructor');
+
+    this.state = {
+      isLoading: false,
+      amount: props.navigation.state.params.amount,
```
[the remark this answers is not in the document]
Comment (Overtorment) [reply screen/send/confirm.js]: nvm if this all was copypasted from send/create

File: screen/send/confirm.js [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+            style={{
+              color: '#37c0a1',
+              fontSize: 14,
+              marginHorizontal: 4,
+              paddingBottom: 6,
+              fontWeight: '500',
+              alignSelf: 'center',
+            }}
+          >
+            {Number(this.state.satoshiPerByte).toFixed(0)} {BitcoinUnit.SATS}
```
Comment (Overtorment) [inline screen/send/confirm.js]: Is that as per  design..?
Better use actual fee, so user knows how much he pays to move funds. satoshiPerByte is a technical metric which shows how profitable it is for miner to include tx in block.
Also, it doesnt day anywhere that this is Fee, its not obvious. Can we write "Fee: xxx"  ?

Comment (Overtorment) [reply screen/send/confirm.js]: same for send/success

File: screen/send/details.js (hunk 2 above)
Comment (marcosrdz) [reply screen/send/details.js]: Done.

File: screen/send/confirm.js [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+let loc = require('../../loc');
+let EV = require('../../events');
+
+export default class Confirm extends Component {
+  static navigationOptions = {
+    headerStyle: {
+      backgroundColor: '#FFFFFF',
+      borderBottomWidth: 0,
+    },
+    headerTintColor: '#0c2550',
```
[the remark this answers is not in the document]
Comment (marcosrdz) [reply screen/send/confirm.js]: Yup. This is definitely something that needs to be done.

**Pull request merged**